src/js/containers/ValidationProgress.jsx

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux'
+import ValidationProgress from '../components/ValidationProgress.jsx'
+
+function mapStateToProps(state) {
+  const {
+    submission
+  } = state.app.submission || {
+    subimission: {
+      id: 1,
+      status: {
+        code: 3,
```

Comment:
3 because the validation progress component is first rendered when the upload is complete, meaning this is a semi-sane default.